src/main/java/goblintserver/GoblintServer.java

```diff
@@ -61,12 +62,11 @@ public String getGoblintSocket() {
     /**
      * Method for constructing the command to run Goblint server.
      * Files to analyse must be defined in goblint conf.
-     *
-     * @throws GobPieException when running Goblint failed.
      */
     private String[] constructGoblintRunCommand() {
         return new String[]{
                 configuration.getGoblintExecutable(),
+                "--enable", "exp.arg",
```

Comment:
The abstract debugger should be enabled/disabled using a GobPie configuration option (maybe disabled by default for now?).
Ideally the ARG construction shouldn't affect the analysis, but in some cases it does cause issues, so silently enabling it for the user isn't too wise.

Reply:
I have added a config option to disable abstract debugging altogether. This has a couple of caveats:
* VS Code shows the abstract debugger in the list of debuggers even if it is disabled. I don't think conditionally disabling debuggers is supported by VS Code (debuggers are at least partly declared in `package.json`). I have edited the error message shown when starting debugging to inform the user of the config option.
* 'show arg' shows up even if abstract debugging is disabled, but gives an error if ARG generation is not enabled separately. I prefer to keep it this way, because a user might want to view the ARG with abstract debugging disabled but `exp.arg` enabled in `goblint.json`.